src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,11 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+
+        for (int i = 0; i < 3; i++) {
```

Comment:
Consider replacing the magic number `3` with a constant. This will improve the readability and maintainability of your code. For example, you could define a constant `private static final int NUMBER_OF_BALLS = 3;` and use it in the loop condition.